buildSrc/src/main/kotlin/nessie-conventions.gradle.kts

```diff
@@ -49,6 +49,7 @@ plugins {
   id("org.projectnessie.buildsupport.spotless")
   id("org.projectnessie.buildsupport.errorprone")
   id("org.projectnessie.buildsupport.publishing")
+  `eclipse`
```

Comment:
just wondering: what does adding this plugin here do for us?

Reply:
It generates the Eclipse files per project (during a `./gradlew eclipse`)